site_root/input/resources/search/SearchParameter-related-organization-organization.json

```diff
@@ -0,0 +1,21 @@
+{
+  "resourceType": "SearchParameter",
+  "id": "related-organization-organization",
+  "url": "http://fhir.kids-first.io/SearchParameter/related-organization-organization",
+  "version": "0.1.0",
+  "name": "related-organization-organization",
+  "status": "active",
+  "experimental": true,
+  "publisher": "Kids First DRC",
+  "description": "Searches Kids First DRC ResearchStudies by organization of related organization.",
+  "code": "related-organization-organization",
```

Comment:
Hmm I know you're trying to follow a naming convention here but this sp name is redundant. It might be confusing to users to do searches like this: `/ResearchStudy?related-organization-organization=foo`.

Maybe its ok to make an exception and have the id and code equal to `related-organization` and leave the other one as `related-organization-role`?

Reply:
Updated!